packages/compass-e2e-tests/tests/no-network-traffic.test.ts

```diff
@@ -117,7 +115,11 @@ describe('networkTraffic: false / Isolated Edition', function () {
 
     if (
       [...connectTargets].some(
-        (target) => !/^127.0.0.1:|^\[::1\]:/.test(target)
+        // Chromium heuristically detects IPv4-only networks by attempting a UDP connection
+        // to 2001:4860:4860::8888 (the IPv6 address for Google Public DNS).
+        (target) =>
+          !/^127.0.0.1:|^\[::1\]:/.test(target) &&
+          !/^\[2001:4860:4860::8888\]:443$/.test(target)
```

Comment:
And I guess this is something we can't control at all?

Reply:
I haven't found one yet. 
It seems like electron command line switches redirect it with the host rules / host resolver rules.
https://www.electronjs.org/docs/latest/api/command-line-switches#--host-rulesrules 
I'll spend a bit of time looking into that and circle back before merging.

Reply:
Early on in our application code to override the host resolution it looks like we can:
```ts
app.commandLine.appendSwitch('host-resolver-rules', 'MAP 2001:4860:4860::8888 some_address');
```
That will cause all of the network requests to start failing if it doesn't succeed though as I think more of the stack relies on this request for knowing where to send requests. I'm thinking as a fix we could maybe provide users a way to specify this themselves.

I'm going to merge this pr for the other fix and create a follow up ticket and link a todo in the code. I think it still needs a bit of investigation and thought